### PR TITLE
Fix widgets

### PR DESCRIPTION
**Pull request by finetjul**
Different fixes for the widgets.

### CODE REVIEW

File: Sources/Widgets/Representations/PolyLineRepresentation/index.js [hunk 1]
```diff
@@ -45,6 +45,21 @@ function vtkPolyLineRepresentation(publicAPI, model) {
...
+   * Change the line/tube thickness.
+   * @param {number} lineThickness
+   */
+  function applyLineThickness(lineThickness) {
+    let scaledLineThickness = lineThickness;
+    if (publicAPI.getScaleInPixels()) {
+      const center = vtkBoundingBox.getCenter(
+        model.internalPolyData.getBounds()
+      );
+      scaledLineThickness *= publicAPI.getPixelWorldHeightAtCoord(center);
```
Comment (floryst) [inline Sources/Widgets/Representations/PolyLineRepresentation/index.js:58]: Does this also work if the line is not perpendicular to the viewing direction? Seems like the line would look like a perspective-projected cylinder to me.

File: Sources/Interaction/Widgets/AbstractWidget/index.js [hunk 2]
```diff
@@ -54,6 +54,9 @@ function vtkAbstractWidget(publicAPI, model) {
       model.interactor.render();
     }
   };
+
+  publicAPI.isDragable = () =>
```
Comment (floryst) [inline Sources/Interaction/Widgets/AbstractWidget/index.js:58]: "dragable" -> "draggable"

File: Sources/Widgets/Core/AbstractWidgetFactory/api.md [hunk 3]
```diff
@@ -41,6 +41,10 @@ Sets visibility for all associated view widgets.
 
 Sets pickable flag for all associated view widgets.
 
+## setDragable(bool)
```
Comment (floryst) [inline Sources/Widgets/Core/AbstractWidgetFactory/api.md:44]: dragable -> draggable

**Review by floryst — COMMENTED** [review]
I didn't closely examine the animation additions, but from a pass-over it seems reasonable.

I looked around in VTK and saw that they also use the "dragable" spelling. However, I don't see "dragable" as some alternative spelling of "draggable".

File: Sources/Widgets/Representations/PolyLineRepresentation/index.js (hunk 1 above)
Comment (finetjul) [reply Sources/Widgets/Representations/PolyLineRepresentation/index.js:58]: you are correct, it looks like a perspective-projected cylinder in that case.

I do not have anything better to suggest though.

**Comment by finetjul** [comment]
> I looked around in VTK and saw that they also use the "dragable" spelling. However, I don't see "dragable" as some alternative spelling of "draggable".

Problem is, `Rendering/Core/Prop` declares a 'dragable' property. Do you suggest to rename it everywhere ?

**Comment by finetjul** [comment]
I merged the MR, but we can still iterate on those discussions, no problem. I just needed the fixes to be merged for the time being. Thanks.

**Pull request merged**